setup.cfg

```diff
@@ -5,10 +5,14 @@ ignore = E133, E226, E228, N802, N803, N806, N812, N813, N815, N816, W503, E203
 exclude =
   bin,
   doc/conf.py,
-  **/*/__init__.py,
-  **/*/version.py,
+  **/__init__.py,
+  **/version.py,
   tests/.tests
+  tests/test_docs.py
 
 [tool:pytest]
 addopts = --flake8
 flake8-ignore = E133 E226 E228 N802 N803 N806 N812 N813 N815 N816 W503 E203
+
+[isort]
+profile=black
```

Comment:
I assume you are happy with default line lengths? We use 110 in other packages and so configure black and isort to allow 110.

Reply:
For python black and isort seem to default to the flake8 value of 110 characters. But for some reason that didn't carry over to the yaml files.

Reply:
I'm a bit surprised. I needed to configure isort and black to 110 explicitly in middleware use because I didn't think it would pick up the flake8 configuration. Maybe that's changed recently.

Reply:
I definitely have lines that are > 80 chars that black has run on. Maybe not isort though.